### ISSUE TITLE
[Request] Pinch Gestures for Minimize or Maximize

### ISSUE THREAD

**Issue by halfmexican** [harshadgavali/gnome-gesture-improvements#169]
I would love to be able to use 3 finger pinch in and out to maximize and minimize windows. I have this set up on xorg and would love to do the same on wayland

**Comment by MadhiasM** [comment]
Yes that would actually be very helpful. Especially for Touchpads which don't support 4 Finger gestures this would help alot
